### ISSUE TITLE
Discontinue community edition or use the same codebase as pro

### ISSUE THREAD

**Issue by sztomi** [ramboxapp/community-edition#1954]
Maintaining a separate codebase is clearly too much effort (which is absolutely understandable). However, this is not useful for anyone:

![image](https://user-images.githubusercontent.com/240594/47911244-40980c80-de95-11e8-9eb6-951cb9ce7702.png)

(and then the bot closes all of those)

Again, I completely understand that the author has no time or energy to maintain the community edition and wants to focus on pro. We all need to put a roof over our heads and food on the table.

I also remember that the argument for keeping pro closed source was basically security by obscurity. I think the way out of this is to use the same codebase and accept that security is not achieved by obscurity (especially not in an application built on electron and web technologies; there is almost zero effort needed to reverse and modify such an application, compared to a native one).

**Comment by chnbz** [comment]
I agree, the current state is horrible.

**Comment by ontherunvaro** [comment]
Bumping this because otherwise the bot will close it for being stale ¯\\_(ツ)_/¯

**Comment by sztomi** [comment]
Good call, and thanks for linking that. Still a bit bummed about this situation.  I want to emphasize: I have nothing against turning an open source project into something that generates revenue - it's the author's own work and their right to do so. It just seems that in this case it was done in an alienating fashion.

**Comment by ontherunvaro** [comment]
Bumping to prevent this issue from being closed.

**Comment by DBX12** [comment]
Bumping again to prevent the issue from being closed.

**Comment by vraravam** [comment]
ping - to mark it not stale!

**Comment by breyvilas** [comment]
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.

Nope!

**Comment by DBX12** [comment]
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.

![image](https://user-images.githubusercontent.com/6484542/64097666-69f32480-cd54-11e9-90da-8acc62a8f4d0.png)
